src/test/java/bio/terra/app/controller/SnapshotAccessRequestApiControllerTest.java

```diff
@@ -186,6 +192,80 @@ void testApproveSnapshotRequest() throws Exception {
     testUpdateStatus(id, response, APPROVE_ENDPOINT);
   }
 
+  @Test
+  void testGetSnapshotRequestGroupMembers() throws Exception {
+    UUID id = UUID.randomUUID();
+    UUID createdId = UUID.randomUUID();
+    SnapshotAccessRequestResponse accessRequest =
+        new SnapshotAccessRequestResponse().createdSnapshotId(createdId);
+    SnapshotAccessRequestMembersResponse expectedResponse =
+        new SnapshotAccessRequestMembersResponse();
+    when(snapshotBuilderService.getRequest(id)).thenReturn(accessRequest);
+    when(snapshotBuilderService.getGroupMembers(id)).thenReturn(expectedResponse);
+    String actualJson =
+        mvc.perform(get(MEMBERS_ENDPOINT, id))
+            .andExpect(status().isOk())
+            .andReturn()
+            .getResponse()
+            .getContentAsString();
+    SnapshotAccessRequestMembersResponse actual =
+        TestUtils.mapFromJson(actualJson, SnapshotAccessRequestMembersResponse.class);
+    assertThat("The method returned the expected response", actual, equalTo(expectedResponse));
+    verifyAuthorization(IamResourceType.SNAPSHOT_BUILDER_REQUEST, id, IamAction.GET);
+  }
```

Comment:
tests look good! maybe you could add a negative test cases to check for the validation that a snapshot must have been created from the given request id

Reply:
Moved that validation to the service code, so I've removed the controller tests for this case